src/mongoose_metrics.erl

```diff
@@ -328,7 +328,12 @@ ensure_metric(Host, Metric, Type, ShortType) when is_list(Metric) ->
     PrefixedMetric = name_by_all_metrics_are_global(Host, Metric),
     case exometer:info(PrefixedMetric, type) of
         ShortType -> {ok, already_present};
-        undefined -> exometer:new(PrefixedMetric, Type)
+        undefined ->
+            case catch exometer:new(PrefixedMetric, Type) of
```

Comment:
`try...catch`, please. There is an example above in my comment.

Reply:
which can be added when we would refactor the whole mongoose_metrics module.

Reply:
OK, will be a part of Metrics Refactoring